docs/framework/migration-guide/retargeting/dbparameterprecision_and_dbparameterscale_are_now_public_virtual_members.md

```diff
@@ -0,0 +1,11 @@
+### DbParameter.Precision and DbParameter.Scale are now public virtual members
+
+|   |   |
+|---|---|
+|Details|<xref:System.Data.Common.DbParameter.Precision> and <xref:System.Data.Common.DbParameter.Scale> are implemented as public virtual properties. They replace the corresponding explicit interface implementations, <xref:System.Data.Common.DbParameter.System#Data#IDbDataParameter#Precision> and <xref:System.Data.Common.DbParameter.System#Data#IDbDataParameter#Scale>.|
```

Comment:
`#` needs to be encoded in xref links. For more info, see https://github.com/dotnet/docs/blob/master/styleguide/template.md#links-to-apis